src/rcore.c

```diff
@@ -3457,6 +3457,29 @@ void OpenURL(const char *url)
 #endif
 #if defined(PLATFORM_WEB)
         emscripten_run_script(TextFormat("window.open('%s', '_blank')", url));
+#endif
+#if defined(PLATFORM_ANDROID)
+        JNIEnv *env;
+        JavaVM *vm = CORE.Android.app->activity->vm;
+        (*vm)->AttachCurrentThread(vm, &env, NULL);
+
+        jstring url_string = (*env)->NewStringUTF(env, url);
```

Comment:
Please, could you follow [raylib naming conventions](https://github.com/raysan5/raylib/blob/master/CONVENTIONS.md) when naming variables on raylib code? It should be `urlString`. Same for the following code.